README.md

```diff
@@ -210,6 +264,10 @@ use `.\` preceding the path to the template. E.g.
     packer validate spel/minimal-linux.pkr.hcl
     ```
 
+    Validation _may_ fail on some newer versions of Packer. The Packer HCL file
+    shipped with this project is known to work with Packer versions up through
+    1.8.7.
```

Comment:
I don't really want to include this. We're likely to update things to work with packer 1.9 in the next couple months or so, and updating this spot in the readme is likely to be missed. I'd rather just add a `packer` block that restricts the version to `< 1.9`. See: https://developer.hashicorp.com/packer/docs/templates/hcl_templates/blocks/packer#packer-block-syntax

Reply:
Updated the note so that it _mentions_ version-compatibility in a future-proof way and requests the reader open an issue if they encounter.